src/status_im2/contexts/add_new_contact/events.cljs

```diff
@@ -0,0 +1,96 @@
+(ns status-im2.contexts.add-new-contact.events
```

Comment:
seems like now this namespace is not required anywhere

Reply:
Yes, I understand what you were asking now. Fixed it. Thanks!